spyt-package/src/main/python/spyt/standalone.py

```diff
@@ -273,9 +272,18 @@ def abort_spark_operations(spark_discovery, client):
         raise error
 
 
-def get_base_cluster_config(global_conf, spark_cluster_version, params, base_discovery_path=None, client=None):
+def get_spyt_distributions(client, params=None) -> SpytDistributions:
+    if params is None:
+        params = {}
```

Comment:
I think we should explicitly resolve yt_root parameter here and pass it to SpytDistributions constructor so it will be more readable